Week1/homework/exercise6.js

```diff
@@ -0,0 +1,7 @@
+var items = [];
+console.log ("An array with more than one value");
+console.log (items);
+var favanimals = ["dog", "giraffe", "horse"];
+console.log (favanimals);
+favanimals.push("Piglet");
```

Comment:
Couple things here again similar to before:
- `const` or `let` instead of `var` keyword.
- If a variable name contains multiple words such as `favanimals` in this case, it is better to use `camelCasing` while naming them. Thus, `favAnimals` would be a better way to name the variable as compared to `favanimals`.